packages/Python/lldbsuite/test/lang/swift/protocol_optional/TestSwiftProtocolOptional.py

```diff
@@ -0,0 +1,16 @@
+# TestSwiftProtocolOptional.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+import lldbsuite.test.lldbinline as lldbinline
+from lldbsuite.test.decorators import *
+
+lldbinline.MakeInlineTest(__file__, globals(),
+                          decorators=[swiftTest,skipUnlessDarwin])
```

Comment:
What is Darwin-specific about this test?

Reply:
like, nothing. Cut&paste failure.

Reply:
Should be fixed now.